projects/igniteui-angular/src/lib/test-utils/tree-grid-functions.spec.ts

```diff
@@ -64,6 +64,12 @@ export class TreeGridFunctions {
         return headerCell;
     }
 
+    public static getHeaderCellMultiColHeaders(fix, columnKey) {
+        const headerCells = fix.debugElement.queryAll(By.css('igx-grid-header'));
+        const headerCell = headerCells.filter((cell) => cell.nativeElement.textContent.indexOf(columnKey) !== -1).pop();
```

Comment:
Using pop() will return  the last element in the array, which might not always return the correct headerCell in the context of multiple header cells within a column group. If we want to get the left-most for example, we can make sure of that by referring to the element with the left-most coordinates.